x/rollapp/keeper/grpc_query_rollapp_test.go

```diff
@@ -15,15 +14,17 @@ import (
 	"golang.org/x/exp/slices"
 	"pgregory.net/rapid"
 
-	keepertest "github.com/dymensionxyz/dymension/v3/testutil/keeper"
 	"github.com/dymensionxyz/dymension/v3/x/rollapp/types"
 )
 
-func TestQuery(t *testing.T) {
+func (suite *RollappTestSuite) TestQuery() {
 	_ = flag.Set("rapid.checks", "50")
 	_ = flag.Set("rapid.steps", "50")
-	rapid.Check(t, func(r *rapid.T) {
-		k, ctx := keepertest.RollappKeeper(t)
+
+	rapid.Check(suite.T(), func(r *rapid.T) {
+		suite.SetupTest()
+
+		k, ctx := suite.App.RollappKeeper, suite.Ctx
 		ids := rapid.SampledFrom([]string{
```

Comment:
this test is neither long running nor does it run when you run other tests, so no need to change

Reply:
done